testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -350,9 +353,8 @@ protected List<String> removeUnsupportedColumnsForTrino(List<String> columns, St
         // TODO: assert exceptions being thrown for each column
         Map<ColumnContext, String> expectedExceptions = expectedExceptionsWithTrinoContext();
 
-        String hiveVersion = getHiveVersionMajor() + "." + getHiveVersionMinor();
         Set<String> unsupportedColumns = expectedExceptions.keySet().stream()
-                .filter(context -> context.hiveVersion().orElseThrow().equals(hiveVersion) && tableName.contains(context.format()))
```

Comment:
why removed?

Reply:
For Trino side failures we don't need check on hive version right ?

Reply:
unless the failure is related to what hive version did modify the version

anyway, why are you changing this line to first add `.orElseThrow()` and then to remove the condition if all you need is remove the condition?

Reply:
With the latest change by squashing the commits - this changeset got disappeared.